src/app/[locale]/viewer/_components/SubmodelsOverviewCard.tsx

```diff
@@ -75,12 +79,14 @@ export function SubmodelsOverviewCard(props: SubmodelsOverviewCardProps) {
                     const endpoint = submodelDescriptor?.endpoints[0].protocolInformation.href;
 
                     if (endpoint) {
-                        const submodelData = await getSubmodelFromSubmodelDescriptor(endpoint);
-                        return {
-                            id: submodelDescriptor.id,
-                            label: submodelDescriptor.idShort ?? '',
-                            submodelData: submodelData,
-                        };
+                        const submodelResponse = await getSubmodelFromSubmodelDescriptor(endpoint);
```

Comment:
Although this change is technically sound, the whole code here doenst make sense to me: Why do we even perform getSubmodelFromSubmodelDescriptor() if we dont use the result afterwards in the return?